lib/seahorn/BvOpSem2Allocators.cc

```diff
@@ -1,5 +1,5 @@
 #include "BvOpSem2Allocators.hh"
-#include "BvOpSem2Context.hh"
+#include "seahorn/BvOpSem2Context.hh"
```

Comment:
the inclusion used to be relative because we do not expose these classes to clients.